server/methods/addAllUserToRoom.js

```diff
@@ -0,0 +1,40 @@
+Meteor.methods({
+  addAllUserToRoom: function(rid) {
+    var now, room, users;
+    var userCount = RocketChat.models.Users.find().count();
+    if(userCount > 500) {
+      throw new Meteor.Error('error-user-limit-exceeded', 'User Limit Exceeded',{
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2777659474/issues/source?bid=3199634&fileBranchId=3362492#l6)